### DISCUSSION TITLE
关于可用的稳定版本

### DISCUSSION THREAD

**Discussion by kikoshoung** [opentiny/tiny-engine#641] [Q&A] [answered]
看到里程碑离得计划是24年7月1日会推出v1.0.0版本，但是项目的tag还是beta版，请问何时可以推出比较稳定的版本呢？

**Comment by hexqi** [comment, marked as answer]
今年5月下旬时候我们启动了项目重构计划，基于新的架构提升项目可扩展能力，并解决当前的一些痛点问题，这项工作预计到9月完成，基于重构版本会在930前先发布rc版本，正式版本发布大概在1030左右。
